### PR TITLE
x11-wm/cwm: version bump

### PR DESCRIPTION
**Pull request by epsilon-0**
Package-Manager: Portage-2.3.99, Repoman-2.3.22
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

File: x11-wm/cwm/cwm-6.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+LICENSE="ISC"
+SLOT="0"
+
+DEPEND="x11-libs/libXft
+	x11-libs/libXinerama
+	x11-libs/libXrandr
+"
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}
```
Comment (a17r) [inline x11-wm/cwm/cwm-6.7.ebuild:29]: this does not make sense.

Comment (epsilon-0) [reply x11-wm/cwm/cwm-6.7.ebuild]: eesh, knew I was forgetting something!  
Thanks for catching that :smile_cat:

File: x11-wm/cwm/cwm-9999.ebuild [hunk 2]
```diff
@@ -25,8 +25,8 @@ RDEPEND="x11-libs/libXft
 	x11-libs/libXinerama
 	x11-libs/libXrandr
 "
-DEPEND="${RDEPEND}"
-BDEPEND="sys-devel/bison
+BDEPEND="${RDEPEND}
```
Comment (DarthGandalf) [inline x11-wm/cwm/cwm-9999.ebuild:28]: Why did you remove DEPEND?

Comment (epsilon-0) [reply x11-wm/cwm/cwm-9999.ebuild:28]: I thought having RDEPEND and BDEPEND should be enough yes?

Comment (DarthGandalf) [reply x11-wm/cwm/cwm-9999.ebuild:28]: It's not. And BDEPEND is for a different architecture.
https://devmanual.gentoo.org/general-concepts/dependencies/index.html

Comment (epsilon-0) [reply x11-wm/cwm/cwm-9999.ebuild]: Ah thanks a lot. That was very informative.  
I've removed RDEPEND and replaced it with just DEPEND, which should look correct now :smile_cat: .

Comment (DarthGandalf) [reply x11-wm/cwm/cwm-9999.ebuild]: Nope.
1. You need all 3. You can skip RDEPEND only if the binary package can be installed on a completely empty system without anything installed.
2. Don't mix DEPEND with BDEPEND, think of cross-compilation.

Comment (epsilon-0) [reply x11-wm/cwm/cwm-9999.ebuild]: ah, I reverted it back to what it was there previously, albeit exchanging DEPEND and RDEPEND for clarity.

File: x11-wm/cwm/cwm-5.6.ebuild [hunk 3]
```diff
@@ -1,42 +0,0 @@
-# Copyright 1999-2018 Gentoo Authors
```
Comment (DarthGandalf) [inline x11-wm/cwm/cwm-5.6.ebuild:1]: You're removing last stable version?

Comment (epsilon-0) [reply x11-wm/cwm/cwm-5.6.ebuild:1]: Yes, I don't see the point in supporting an ebuild with memory leaks and upstream has deemed dead for > 4 years

Comment (DarthGandalf) [reply x11-wm/cwm/cwm-5.6.ebuild:1]: I suggest stabilizing the new version first. Either 6.6 or 6.7

https://devmanual.gentoo.org/ebuild-maintenance/removal/index.html

Comment (epsilon-0) [reply x11-wm/cwm/cwm-5.6.ebuild:1]: 6.6 stabilize sounds good.

**Comment by epsilon-0** [comment]
thanks!
